src/index.spec.ts

```diff
@@ -1205,7 +1214,7 @@ describe('EppoClient config', () => {
     const retryManager = eventDispatcher['retryManager'];
     const batchProcessor = eventDispatcher['batchProcessor'];
     expect(eventDispatcher['deliveryIntervalMs']).toEqual(1);
-    expect(batchProcessor['batchSize']).toEqual(5);
+    expect(batchProcessor['batchSize']).toEqual(100);
```

Comment:
cc @felipecsl I had to change this because 5 is below the minimum allowed batch size

https://github.com/Eppo-exp/js-sdk-common/blob/b1048a32587a809bed351af07185592c9f774150/src/events/batch-event-processor.ts#L4-L13